### PR TITLE
fix: validate README.md

### PR DESCRIPTION
**Pull request by dgarus**
Part of https://github.com/artipie/artipie/issues/1031

- Corrected and extended readme file
- added codecov report

### CODE REVIEW

**Comment by dgarus** [comment]
@olenagerasimova could you review, please?
I don't have permissions for a review request here.

File: README.md [hunk 1]
```diff
@@ -4,9 +4,9 @@
 [![DevOps By Rultor.com](http://www.rultor.com/b/artipie/vertx-server)](http://www.rultor.com/p/artipie/vertx-server)
 [![We recommend IntelliJ IDEA](https://www.elegantobjects.org/intellij-idea.svg)](https://www.jetbrains.com/idea/)
 
-[![Build Status](https://img.shields.io/travis/artipie/vertx-server/master.svg)](https://travis-ci.org/artipie/vertx-server)
 [![Javadoc](http://www.javadoc.io/badge/com.artipie/vertx-server.svg)](http://www.javadoc.io/doc/com.artipie/vertx-server)
 [![License](https://img.shields.io/badge/license-MIT-green.svg)](https://github.com/artipie/vertx-server/blob/master/LICENSE)
+[![codecov](https://codecov.io/gh/artipie/vertx-server/branch/master/graph/badge.svg)](https://codecov.io/gh/artipie/vertx-server)
 [![Hits-of-Code](https://hitsofcode.com/github/artipie/vertx-server)](https://hitsofcode.com/view/github/artipie/vertx-server)
 [![Maven Central](https://img.shields.io/maven-central/v/com.artipie/vertx-server.svg)](https://maven-badges.herokuapp.com/maven-central/com.artipie/vertx-server)
 [![PDD status](http://www.0pdd.com/svg?name=artipie/vertx-server)](http://www.0pdd.com/p?name=artipie/vertx-server)
```
Comment (olenagerasimova) [inline README.md:12]: I'd suggest to add some simple explanation: what is `vertx-server`, what can it do and how we use it in artipie (IT for adapters and as http server in artipie/artipie)

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@dgarus thanks, one suggestion

**Comment by dgarus** [comment]
@olenagerasimova 
added a sentence what `vertx-server` is, could you please have a look?

**Review by olenagerasimova — APPROVED** [review]
@dgarus thanks, merging

**Pull request merged**